tests/test_code_format.py

```diff
@@ -18,9 +18,6 @@ def test_flake8():
     cmd.extend(['--ignore=C,D,Q,I'])
     # work around for https://gitlab.com/pycqa/flake8/issues/179
     cmd.extend(['--jobs', '1'])
-    if sys.version_info < (3, 4):
-        # Unless Python3, skip files with new syntax, like `yield from`
-        cmd.append('--exclude={0}/*async_execute_process_asyncio/impl.py'.format(source_dir))
```

Comment:
@wjwwood said:

> These files would likely no longer be needed. And some of the files could be collapsed once there's only one implementation.

But I think you were thinking of this backwards.  We do need the `asyncio` implementation here, but in a Python 2 world we had to exclude those files from being flake8 checked.  Now that we are removing Python 2 support, we no longer need that exclusion.

But please correct me if I misunderstood you.